tests/Microsoft.Identity.Test.Integration.netfx/HeadlessTests/RuntimeBrokerTests.cs

```diff
@@ -46,7 +46,7 @@ public async Task WamSilentAuthUserInteractionRequiredAsync()
                .Create("04f0c124-f2bc-4f59-8241-bf6df9866bbd")
                .WithAuthority("https://login.microsoftonline.com/organizations");
 
-            IPublicClientApplication pca = pcaBuilder.WithBrokerPreview().Build();
+            IPublicClientApplication pca = pcaBuilder.WithWindowsBroker().Build();
```

Comment:
Can you also add a integration test for the Desktop Broker calling into Runtime Broker?

Reply:
That is static method call. So, I would avoid writing test for it. If you have suggestions, please let me know.